src/components/title/_fields/package-select/package-select-field.js

```diff
@@ -35,13 +33,13 @@ const PackageSelectField = ({
   };
 
   return (
-    <div
-      data-test-eholdings-package-select-field
-      className={css.packageSelectFieldContainer}
-    >
+    <div data-test-eholdings-package-select-field>
       <Field
         name="packageId"
         component={Selection}
+        popper={{
+          portal: document.getElementById('ModuleContainer'),
```

Comment:
I think the root cause was the `overflow` on the modal, and the fix works when portal renders to `OverlayContainer`. So I'd prefer to use `OverlayContainer` because it's usually the element where we render such dropdowns
Good catch with the overflow!

Reply:
Scrolling doesn't work for me with `OverlayContainer`.